pytest/tests/mocknet/helpers/neard_runner.py

```diff
@@ -141,6 +143,77 @@ class TestState(Enum):
 }"""
 
 
+class EpochConfigOverrides:
+
+    def __init__(self, epoch_config_overrides):
+        self.common_overrides = ""
+        if 'all' in epoch_config_overrides:
+            self.common_overrides = epoch_config_overrides['all']
+            del epoch_config_overrides['all']
+        self.epoch_config_overrides = epoch_config_overrides
+        self.new_protocol_versions = list(epoch_config_overrides.keys())
+
+    def override_file(self, epoch_config_path, overrides):
+        with open(epoch_config_path, 'r') as f:
+            json_data = json.load(f)
+
+        # Apply the overrides
+        updated_json = jq.compile(overrides).input(json_data).first()
```

Comment:
Instead of jq, did you consider just having the `--epoch-config-overrides` flag take a path to a json file where the wanted changes are? It might be a bit easier if someone wants to make lots of edits (imagine editing the fields you included in the PR description, along with editing `shard_layout` and `num_block_producer_seats_per_shard` or something). In that case a command line arg with a bunch of big jq programs might be kind of unwieldy